umake/frameworks/android.py

```diff
@@ -76,12 +76,12 @@ def parse_download_link(self, tag, line, in_download):
 
 class AndroidStudio(umake.frameworks.baseinstaller.BaseInstaller):
 
-    def __init__(self, category):
+    def __init__(self, category, force_loading):
```

Comment:
Instead of passing this parameter (now that we have more than one) in each frameworks, I would suggest to replace the signature and call with with:
```
def __init__(self, **kwargs):
    super().__init__(name=…
                              only_on_archs=…
                              **kwargs)
```
and, so, you remove as well the category=category duplication in each framework at the same time!

Do not forget also to modify the tests frameworks which are under `tests/data/*frameworks/`